tfimm/layers/lora.py

```diff
@@ -0,0 +1,92 @@
+import tensorflow as tf
+from dataclasses import dataclass
+from tfimm.architectures import convnext
+
+
+class LoraDense(tf.keras.layers.Dense):
+    def __init__(self, *args, lora_rank: int = 4, lora_alpha: float = 1, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.lora_rank = lora_rank
+        self.lora_alpha = lora_alpha
+        self.scaling = lora_alpha / lora_rank
```

Comment:
I would add in `__init__` the lines
```
self.kernel_lora_a = None
self.kernel_lora_b = None
```
otherwise PyCharm complains that we are defining all instance attributes in `__init__`.

Reply:
done